app/src/test/java/pm/gnosis/heimdall/ui/transactions/details/AssetTransferTransactionDetailsViewModelTest.kt

```diff
@@ -0,0 +1,427 @@
+package pm.gnosis.heimdall.ui.transactions.details
+
+import android.content.Context
+import io.reactivex.Observable
+import io.reactivex.Single
+import io.reactivex.observers.TestObserver
+import io.reactivex.subjects.PublishSubject
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.BDDMockito.*
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.MockitoJUnitRunner
+import pm.gnosis.heimdall.StandardToken
+import pm.gnosis.heimdall.common.utils.DataResult
+import pm.gnosis.heimdall.common.utils.ErrorResult
+import pm.gnosis.heimdall.common.utils.Result
+import pm.gnosis.heimdall.data.repositories.*
+import pm.gnosis.heimdall.data.repositories.models.ERC20Token
+import pm.gnosis.heimdall.data.repositories.models.ERC20Token.Companion.ETHER_TOKEN
+import pm.gnosis.heimdall.data.repositories.models.ERC20TokenWithBalance
+import pm.gnosis.heimdall.ui.transactions.details.AssetTransferTransactionDetailsContract.*
+import pm.gnosis.heimdall.ui.transactions.details.AssetTransferTransactionDetailsFragment.TransactionInputException
+import pm.gnosis.heimdall.ui.transactions.details.AssetTransferTransactionDetailsFragment.TransactionInputException.Companion.AMOUNT_FIELD
+import pm.gnosis.heimdall.ui.transactions.details.AssetTransferTransactionDetailsFragment.TransactionInputException.Companion.TOKEN_FIELD
+import pm.gnosis.heimdall.ui.transactions.details.AssetTransferTransactionDetailsFragment.TransactionInputException.Companion.TO_FIELD
+import pm.gnosis.model.Solidity
+import pm.gnosis.models.Transaction
+import pm.gnosis.models.Wei
+import pm.gnosis.tests.utils.ImmediateSchedulersRule
+import pm.gnosis.tests.utils.MockUtils
+import pm.gnosis.tests.utils.mockGetString
+import java.math.BigInteger
+
+@RunWith(MockitoJUnitRunner::class)
+class AssetTransferTransactionDetailsViewModelTest {
+    @JvmField
+    @Rule
+    val rule = ImmediateSchedulersRule()
+
+    @Mock
+    lateinit var detailsRepository: TransactionDetailsRepository
+
+    @Mock
+    lateinit var tokenRepository: TokenRepository
+
+    private lateinit var viewModel: AssetTransferTransactionDetailsViewModel
+
+    @Before
+    fun setUp() {
+        viewModel = AssetTransferTransactionDetailsViewModel(detailsRepository, tokenRepository)
+    }
+
+    @Test
+    fun loadFormDataNoTransaction() {
+        val testObserver = TestObserver<FormData>()
+        viewModel.loadFormData(null).subscribe(testObserver)
+
+        testObserver.assertResult(FormData())
+        Mockito.verifyNoMoreInteractions(detailsRepository)
+        Mockito.verifyNoMoreInteractions(tokenRepository)
+    }
```

Comment:
Why not use `BDDMockito`? (applies to the other tests)

Reply:
Because AS automatically changed it to mockito ... Don't ask me why